api/v1alpha1/ocivalidator_types.go

```diff
@@ -52,17 +47,36 @@ type OciRegistryRule struct {
 }
 
 func (r OciRegistryRule) Name() string {
-	return fmt.Sprintf("%s/%s", r.Host, r.Artifacts)
+	var artifacts strings.Builder
+	artifacts.WriteString("[")
+	l := len(r.Artifacts)
+	for i, a := range r.Artifacts {
+		artifacts.WriteString(a.Ref)
+		if i < l-1 {
+			artifacts.WriteString(", ")
+		}
+	}
+	artifacts.WriteString("]")
```

Comment:
I'm a bit concerned that this could get out of hand if the rule includes a large # of artifacts. WDYT about just displaying the number of artifacts after the host name?

Reply:
For example, our airgap bundles will include >50 images

Reply:
that works, my only concern would be the case where a user for whatever reason creates 2 different OciRegistryRules pointing to the same registry and specifying the same number of artifacts in each rule, it would be tough to distinguish to two apart.

but now that i type that out, that feels more like the user is setting things up wrong